search-sentiments-map/public/scripts/map-script.js

```diff
@@ -51,7 +51,7 @@ function initMap() {
   map.data.setStyle(styleFeature);
   map.data.addListener('mouseover', mouseInToRegion);
   map.data.addListener('mouseout', mouseOutOfRegion);
-  map.data.addListener('click', clickOnRegion);
+  map.data.addListener('click', onClickCountry);
```

Comment:
The reason I didn't name it specifically country was for when we added the zoom to US states level, I thought it would use the same function.

Reply:
but it actually has a lot of "country" in the function

Reply:
We should probly use a different function for states

Reply:
I mean its just variable names that can be renamed to regionId, regionName etc...because the states will have this same data.  But if you want to split them into two functions we can...its just they would be identical, so we could refactor it by using the same function.

Reply:
I mean the states won't have the same search results

Reply:
Yeah but we could have a small if else for whether we are in state/country mode and call the correct show results functions...but a lot of the start is the same? Getting the variables etc.